src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  function addRobot(firstValue) {
+    callCount++;
+
+    const errorMessage = `Bzzz... Error!`;
+
+    return (secondValue) => callCount < 4 || callCount % 2 !== 0
+      ? firstValue + secondValue
+      : errorMessage;
```

Comment:
I'd rather use if-statement, also, I need to inverse it
```
if (...) { // if something goes wrong
	// we stop doing what we're doing
}

// otherwise – we do what we need
```
this pattern looks more appropriate